core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsGroupComponent.scala

```diff
@@ -241,15 +241,10 @@ trait RawlsGroupComponent {
       }
     }
 
-    def intersectGroupMembership(group1: RawlsGroupRef, group2: RawlsGroupRef): ReadAction[Set[RawlsUserRef]] = {
-      val group1Members = flattenGroupMembership(group1)
-      val group2Members = flattenGroupMembership(group2)
+    def intersectGroupMembership(groups: Set[RawlsGroupRef]): ReadAction[Set[RawlsUserRef]] = {
+      val flattenedGroups = DBIO.sequence(groups.map(flattenGroupMembership).toSeq)
```

Comment:
I added a DBIO.sequence here, but it's arguably no worse than what was already here (it was just hardcoded to only handle 2 groups)

Question: Worth the effort to investigate how to do this in batch?